src/raftstore/store/worker/apply.rs

```diff
@@ -195,48 +194,131 @@ impl ApplyCallback {
 
 struct ApplyContext<'a> {
     host: &'a CoprocessorHost,
-    wb: WriteBatch,
+    wb: Option<WriteBatch>,
```

Comment:
What's the benefit of using an `Option` here?

Reply:
To be able take the wb without drop context or creating a new `WriteBatch`.